explorer/src/Pos/Explorer/Txp/Global.hs

```diff
@@ -72,7 +72,7 @@ applyBlund txpBlund = do
             Left genesisBlock -> SlotId
                                   { siEpoch = genesisBlock ^. epochIndexL
                                   , siSlot  = minBound
-                                  -- ^ Genesis block doesn't have a slot, set to minBound
+                                  -- Genesis block doesn't have a slot, set to minBound
```

Comment:
This is the correct way to go, bravo. This is a comment the programmer left to describe a subtleties of the field, we don't want it to be exposed in the docs.